hack/e2e/aks-addon.sh

```diff
@@ -88,6 +88,10 @@ if [ "$E2E_TARGET" = "pr" ]; then
 
 fi
 
+az acr import --name ${ACR_NAME} --source docker.io/library/alpine:latest
```

Comment:
The current e2e workflow will create ACR only if we run it for PR. If the e2e tests run on MCR, ACR won't be created as we are using the MCR images. If you find it necessary to have ACR now, we can remove the`E2E_TARGET` param for the whole scenario.

Reply:
We need to create ACR since this feature is specifically for adding support to pull images from ACR using kubelet-identity, without having to use username/password credentials.

Reply:
Then let's have one ACR instead of 2.